active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,662 @@
+[ContentPropertyAttribute](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Markup.ContentPropertyAttribute).
+
+In a UWP app any UI thread already has a Window on it, which you can retrieve using the static 
+[Window.Current](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Window.Current) 
+property. You can create additional windows by creating additional 
+[CoreApplicationViews](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Core.CoreApplicationView), 
+which are always created on a new thread, and which automatically create the following for the new thread: 
+[ApplicationView](https://docs.microsoft.com/uwp/api/Windows.UI.ViewManagement.ApplicationView), 
+[CoreWindow](https://docs.microsoft.com/uwp/api/Windows.UI.Core.CoreWindow), 
+and the Xaml [Window](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Window).
+
+# Create a new Window on a new thread
+
+This creates a new Xaml Window on a new thread in a UWP app:
+
+```CS
+_ = CoreApplication.CreateNewView().DispatcherQueue.TryEnqueue(() =>
+{
+    // This code runs on the new thread
+    Window.Current.Content = new TextBlock() { Text = "Hello" };
+    Window.Current.Activate();
+});
+```
+
+To create a new Window on a new thread in a Desktop app, create the thread first:
+
+```CS
+var thread = new Thread(() =>
+{
+    Window window = new Window();
+    window.Content = new TextBlock() { Text = "Hello" };
+    window.Activate();
+    window.Closed += (_, __) => w.DispatcherQueue.InvokeShutdown();
+
+    var syncContext = new DispatcherQueueSyncronizationContext();
+    SynchronizationContext.SetSynchronizationContext(syncContext);
+
+    this.ProcessEvents(); // Message pump
+});
+
+thread.SetApartmentState(ApartmentState.STA);
+thread.Start(); 
+```
+
+> Spec note: The DispatcherQueueSyncronizationContext is a new API 
+[described here](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/winui3/DispatcherQueueUpdates.md)
+
+### Remarks
+
+The Window class can be used both in a UWP app or a Desktop app.
+
+In a Desktop app you can create (`new`) a Window, even if the current thread already has a Window on it.
+In a UWP app you must create a new 
+[CoreApplicationView](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Core.CoreApplicationView),
+which will automatically create a new thread and a Window on it. 
+See the 'Create new Window in UWP application' example.
+
+> Implementation note: Attempting to activate a new window in a UWP app will use 
+[RoOriginateError](https://docs.microsoft.com/en-us/windows/win32/api/roerrorapi/nf-roerrorapi-rooriginateerror) 
+to show an explanatory message in the debugger.
+
+Creating a new Window in a Desktop app creates a new top level HWND.
+
+## Window.ExtendsContentIntoTitleBar
+
+Gets or sets a value that specifies whether the default window title bar should be removed, creating more space for content.
```

Comment:
Rather than saying it makes more space, might it not be better and more accurate to say something like **it allows the App's UI to extend to the top of the window's frame, where the titlebar would usually be.**